source/tasks/Utils/taskInput.ts

```diff
@@ -28,4 +29,8 @@ export class ConcreteTaskWrapper implements TaskWrapper {
     public setOutputVariable(name: string, value: string) {
         tasks.setVariable(name, value, false, true);
     }
+
+    public getVariable(name: string): string | undefined {
```

Comment:
I wonder if this might be clearer if the signature was `getOutputVariable(stepName: string, variableName: string)`?
In essence, it would move the string interpolation from [here](https://github.com/OctopusDeploy/OctoTFS/pull/282/files#diff-dccf221e8e60ab5dfe810d7cf6a22353bcf6a6bde4a3a66a086b5dd270e1c194R24) into this method and I think the method's contract/usage might be clearer.